rmrl/document.py

```diff
@@ -152,17 +153,15 @@ def render_to_painter(self, canvas, vector, template_alpha):
 class DocumentPageLayer:
     pen_widths = []
 
-    def __init__(self, page, name=None):
+    def __init__(self, page, colors, name=None):
         self.page = page
         self.name = name
 
         self.colors = [
-            #QSettings().value('pane/notebooks/export_pdf_blackink'),
-            #QSettings().value('pane/notebooks/export_pdf_grayink'),
-            #QSettings().value('pane/notebooks/export_pdf_whiteink')
-            (0, 0, 0),
-            (0.5, 0.5, 0.5),
-            (1, 1, 1)
+            colors.black.rgb,
+            colors.gray.rgb,
+            colors.white.rgb,
+            colors.highlight.rgb,
```

Comment:
Namedtuples allow position-based indexing, so as long as we get it in the right order, we could just pass `self.colors = colors` here, and then do `colors[i].rgb` in the pen itself.

I'm not completely sold on this idea.  I does decrease the amount of fussing needed here, but it also locks our color structure to the details of the lines files from remarkable.  I could go either way here, so I'll let you make the decision.